src/mapbox_logo.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+var MapboxLogoControl = L.Control.extend({
+
+    options: {
+        position: 'bottomleft',
+    },
+
+    initialize: function(options) {
+        L.Util.setOptions(this, options);
+    },
+
+    onAdd: function(map) {
+        this._container = L.DomUtil.create('div', 'mapbox-logo');
+        L.DomEvent.disableClickPropagation(this._container);
```

Comment:
Probably not needed since it's not a link with attached listeners -- it was copied over from another control.